os/binfmt/binfmt_loadbinary.c

```diff
@@ -78,6 +78,15 @@ int load_binary(FAR const char *filename, size_t binsize, size_t offset)
 		goto errout;
 	}
 
+#ifdef CONFIG_APP_BINARY_SEPARATION
+	/* Allocate the RAM partition to load the app into */
+	uint32_t *start_addr;
+	uint32_t size;
+	struct tcb_s *tcb;
+
+	mm_allocate_ram_partition(&start_addr, &size);
```

Comment:
As I said before in a chat, we can't use equally divided ram partition.
To support different sizes of partition table, you should modify this. What is your plan?

Reply:
@sunghan-chang,
True. This implementation is for fixed size partitions. Let us keep this implementation for now. 
I have a plan for dynamic partitions. I will implement it separately and raise a different PR for it.

Reply:
@kishore-sn Dynamic allocation and different size of partition is different. Anyway can I understand that you will support both with different commits?

Reply:
Yes. We will handle both cases in a new PR.